src/libs/kata-sys-util/src/spec.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
```

Comment:
Should we move this file into kata-types? Some related constants/data structs are defined there.

Reply:
The spec contains some operation functions, so it feels more appropriate to put it in kata-sys-util

Reply:
Seems duplicated with
https://github.com/kata-containers/kata-containers/blob/runtime-rs/src/libs/kata-types/src/k8s.rs#L36

https://github.com/kata-containers/kata-containers/blob/runtime-rs/src/libs/kata-types/src/k8s.rs#L59

Reply:
Thanks to the detailed review, this issue has been fixed.